src/components/contactInfoForm/ContactInfoForm.tsx

```diff
@@ -50,9 +50,14 @@ function ContactInfoForm() {
   }, [reset, initialState]);
 
   useEffect(() => {
-    if (submitSuccess) toast.success("changes saved");
-    //  @ts-ignore
-    if (submitError) toast.error(error.data.message[0]);
+    if (submitSuccess) toast.success(i18next.profileSuccessSubmitMessage);
+    if (submitError) {
+      toast.error(
+        // @ts-ignore
+        error?.data?.message[0] ||
```

Comment:
is it possible to remove @ts-ignore?

Reply:
unfortunately no, I get the errors object from the hook, which is not typed and the typescript throws an error. so I turned off type checking and added a static string in case there is no such value in the object

Reply:
Check sign up component, how its done there. Instead of showing the server message, check for status.
Because @ts-ignore is a serious thing